maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,735 @@
+class MaximEvalConfig:
+    """Maxim eval config."""
+
+    evaluators: list[str]
+    additional_variables: list[dict[str, str]]
+
+    def __init__(self):
+        self.evaluators = []
+        self.additional_variables = []
+
+
+def extract_usage_from_response(response) -> dict:
```

Ruff (0.12.2)

Missing type annotation for function argument `response`

(ANN001)